docs/lib/datastore/fragments/android/data-access.md

```diff
@@ -10,8 +10,8 @@ Post post = Post.builder()
     .build();
 
 Amplify.DataStore.save(post,
-    saved -> Log.i("DataStore", "Saved a post: " + saved.item()),
-    saveFailure -> Log.e("DataStore", "Save failed.", saveFailure)
+    saved -> Log.i("DataStore", "Saved a post."),
```

Comment:
("While you're here" comment.) Consistency, hobgoblin, little minds: why not use the same tag on each log line? I'm not sure I see the value in some being set to "QuickStart" or "GetStarted" or whatever.

Reply:
Agreed, also. Let's go with `"QuickStart"`, uniformly, everywhere.

Reply:
Eh, actually, I settled on `"GetStarted"`, which sounds less gimmicky. `Quick` looks like its about to drop its `c`, and do some telemarketing as a wonder-product.